server/sqlstore/playbook.go

```diff
@@ -846,6 +862,84 @@ func (p *playbookStore) GetAutoFollows(playbookID string) ([]string, error) {
 	return autoFollows, nil
 }
 
+func (p *playbookStore) GetMetric(id string) (*app.PlaybookMetricConfig, error) {
+	metricSelect := p.queryBuilder.
+		Select(
+			"c.ID",
+			"c.PlaybookID",
+			"c.Title",
+			"c.Description",
+			"c.Type",
+			"c.Target",
+		).
+		From("IR_MetricConfig c").
+		Where(sq.Eq{"c.ID": id})
+
+	var metric app.PlaybookMetricConfig
+	err := p.store.getBuilder(p.store.db, &metric, metricSelect)
+	if err != nil {
+		return nil, err
+	}
+
+	return &metric, nil
+}
+
+func (p *playbookStore) AddMetric(playbookID string, config app.PlaybookMetricConfig) error {
+	numExistingMetrics, err := p.GetNumMetrics(playbookID)
+	if err != nil {
+		return err
+	}
+
+	if numExistingMetrics >= app.MaxMetricsPerPlaybook {
+		return errors.Errorf(fmt.Sprintf("playbook cannot have more than %d key metrics", app.MaxMetricsPerPlaybook))
```

Comment:
There's no need for `fmt.Sprintf` here
```suggestion
		return errors.Errorf("playbook cannot have more than %d key metrics", app.MaxMetricsPerPlaybook)
```